### ISSUE TITLE
Questions about training.

### ISSUE THREAD

**Issue by mls1999725** [mks0601/Hand4Whole_RELEASE#13]
Hi, I'm very interested in your excellect work! Could you please tell me the number of epochs in each of the three stages (pre-train Hand4Whole, pre-train hand-only Pose2Pose and combine pre-trained Hand4Whole and hand-only Pose2Pose and fine-tune it), and could it be that there are only 7 epochs in the first stage and only 12 epochs in the second stage as shown in the codes? 

By the way, according to the paper, is the learning rate multiplied by 0.1 at the 10th epoch for all three stages?

Looking forward to your reply!

**Comment by mks0601** [comment]
1. pre-train Hand4Whole: set `lr_dec_epoch` in `config.py` to [4,6] and `end_epoch` in `config.py` to 7. The network should be trained after initializing it with 2D pose estimation network as described in README.
2. pre-train hand-only Pose2Pose: set `lr_dec_epoch` in `config,py` to [10,12] and `end_epoch` in `config.py` to 13.
3. combine pre-trained ones and fine-tune it: set `lr_dec_epoch` in `config.py` to [4,6] and `end_epoch` in `config.py` to 7.  The learning rate should be 1e-5 as described in README.

**Comment by mls1999725** [comment]
Thanks a lot !

**Comment by mls1999725** [comment]
By the way, do I need to switch the codes to the Pose2Pose branch during the second stage of training?
